client/my-sites/themes/index.node.js

```diff
@@ -29,12 +34,13 @@ export default function ( router ) {
 	];
 	router(
 		showcaseRoutes,
-		setHrefLangLinks,
 		ssrSetupLocale,
 		fetchThemeFilters,
 		validateVertical,
 		validateFilters,
 		fetchThemeData,
+		setHrefLangLinks,
```

Comment:
Just to check - `setLocalizedCanonicalUrl` needs to be after applied after `ssrSetupLocale`, and `setHrefLangLinks` middleware has been moved further down as well so that it's next to `setLocalizedCanonicalUrl`. Is that correct and is there anything else that I might be missing?

Reply:
Thanks for reviewing again!

Yes, that's right: `setLocalizedCanonicalUrl` needs to be applied after `ssrSetupLocale`, because it uses `getLocaleSlug`. 🙂 

You're right that `setHrefLangLinks` didn't necessarily have to be moved down (because it doesn't use `getLocaleSlug`), but I kept them together because they're pretty closely related.

Do you think that's OK, or would you prefer a different strategy? Any input is appreciated! 😀

Reply:
> Do you think that's OK, or would you prefer a different strategy? Any input is appreciated! 😀

Yes, I think it's perfectly fine. Just wanted to make sure I'm not missing something :)